src/gt4sd/training_pipelines/tests/test_training_moses_vae.py

```diff
@@ -0,0 +1,68 @@
+"""Moses VAE Trainer unit tests."""
+
+from typing import Any, Dict, cast
+
+import pkg_resources
+
+from gt4sd.training_pipelines import TRAINING_PIPELINE_MAPPING, MosesVAETrainingPipeline
+
+MODEL_ARTIFACTS_LOAD = VALID_FILE_PATH = pkg_resources.resource_filename(
+    "gt4sd",
+    "training_pipelines/tests/guacamol_test_data/",
+)
+
+template_config = {
+    "model_args": {
+        "embedding_size": 32,
+        "q_cell": "gru",
+        "q_bidir": False,
+        "q_d_h": 256,
+        "q_n_layers": 1,
+        "q_dropout": 0.5,
+        "d_cell": "gru",
+        "d_n_layers": 3,
+        "d_dropout": 0,
+        "d_d_h": 512,
+        "d_z": 128,
+        "freeze_embeddings": False,
+    },
+    "training_args": {
+        "n_batch": 512,
+        "grad_clipping": 50,
+        "kl_start": 0,
+        "kl_w_start": 0,
+        "kl_w_end": 0.05,
+        "lr_start": 3 * 1e-4,
+        "lr_n_period": 10,
+        "lr_n_restarts": 10,
+        "lr_n_mult": 1,
+        "lr_end": 3 * 1e-4,
+        "n_last": 1000,
+        "n_jobs": 1,
+        "n_workers": 1,
+    },
+    "common_args": {
```

Comment:
not sure I follow the logic, can't we use the same ones we use for the paccmann-vae tests?

Reply:
I have pushed a change for this, thanks :)